### PR TITLE
Submission for turn one: Random events fire when dynamite goes off.

### PR DESCRIPTION
**Pull request by CALEBPENTECOST**
Add new event, which will attempt to trigger a random (non-infinite, non game-ending) event twice

### CODE REVIEW

File: cogvents.py [hunk 1]
```diff
@@ -54,6 +54,20 @@ def vowelLimiter(data):
...
+  whichEvent = (random.randint(0, len(events)-1))
+  theEvent = list(events.keys())[whichEvent]
+  print(("\tRandomExecution attempting to emit: "  + theEvent))
+  gDict["fnRandomExecution"] = theEvent
+  if theEvent != "Event_Win" and theEvent != "Event_Start" and theEvent != "Event_Boom":
+    Emit(theEvent, whichEvent)
+    Emit(theEvent, whichEvent)
+  else:
+    print("\t\tI aint emitting that!")
+  print(gDict)
```
Comment (bryantfhayes) [inline cogvents.py]: Print the whole dictionary?! You crazy!?

File: cogvents.py [hunk 2]
```diff
@@ -54,6 +54,20 @@ def vowelLimiter(data):
...
 
+def fnRandomExecution(data):
+  import random
+  global gDict
+  whichEvent = (random.randint(0, len(events)-1))
+  theEvent = list(events.keys())[whichEvent]
+  print(("\tRandomExecution attempting to emit: "  + theEvent))
+  gDict["fnRandomExecution"] = theEvent
+  if theEvent != "Event_Win" and theEvent != "Event_Start" and theEvent != "Event_Boom":
+    Emit(theEvent, whichEvent)
```
Comment (bryantfhayes) [inline cogvents.py]: Twice!?

File: readme.txt [hunk 3]
```diff
@@ -11,9 +11,9 @@ Rules:
 9. ONLY EDIT COGVENTS.PY
 
 Turn Order:
-[*] Bryant
+[ ] Bryant
 [ ] Ben
-[ ] Caleb
+[*] Caleb
```
Comment (bryantfhayes) [inline readme.txt]: Nice job!

File: cogvents.py [hunk 4]
```diff
@@ -54,6 +54,20 @@ def vowelLimiter(data):
...
+    Emit(theEvent, whichEvent)
+  else:
+    print("\t\tI aint emitting that!")
+  print(gDict)
+
```
Comment (bryantfhayes) [inline cogvents.py]: Consider returning csys_status? ;)

File: cogvents.py (hunk 2 above)
Comment (CALEBPENTECOST) [inline cogvents.py]: Yessir. Good luck everyone!

File: cogvents.py (hunk 1 above)
Comment (CALEBPENTECOST) [inline cogvents.py]: Only when this event runs, yes.

File: readme.txt (hunk 3 above)
Comment (CALEBPENTECOST) [inline readme.txt]: I'm the only one to pull this off so far.

File: cogvents.py (hunk 4 above)
Comment (CALEBPENTECOST) [inline cogvents.py]: Consider returning to wherever you came from.

File: readme.txt (hunk 3 above)
Comment (bryantfhayes) [inline readme.txt]: Well mine was correct outta the gate

**Comment by CALEBPENTECOST** [comment]
I don't know if the rules allow it, but I'm OK with future changes adding to the list of "do not fire" events, so long as it is in the interest of preventing someone from just randomly being declared the winner.

**Pull request merged**